### PR TITLE
Added postman collections for API tests

### PR DESCRIPTION
**Pull request by karthikmsqa**
# Description

Added postman test collection and data file for the API tests

## Type of change

- [x] New tests(non-breaking change which adds tests)

# How Has This Been Tested?

Tested on postman app and newman locally.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## For bcgov contributors:

This PR is part of the jira ticket: https://justice.gov.bc.ca/jira/browse/MDYNAMICS-406

### CODE REVIEW

File: maltd-frontend/package-lock.json [hunk 1]
```diff
@@ -11687,7 +11686,6 @@
       "version": "9.0.0",
       "resolved": "https://registry.npmjs.org/keycloak-js/-/keycloak-js-9.0.0.tgz",
       "integrity": "sha512-YvxPWKScIrSWCdxfuibTN0HauscsnKJBFhjXk7WAHmcxuEMyR1wRmSjwfxErWlHknJPLNDON5yrK/FDv0CiXlw==",
-      "dev": true,
```
Comment (sdevalapurkar) [inline maltd-frontend/package-lock.json]: not too sure about these changes either....

File: postman-tests/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Tests
+
+## Postman
+
+We maintain a [postman collection](postman-tests/MALT-Api-Search-And-Add-Users.postman_collection.json), [postman environment](malt-local-env.postman_environment.json) and an external datafile with valid/invalid data to pass on the requests.
```
Comment (sdevalapurkar) [inline postman-tests/README.md]: nit:
```suggestion
We maintain a [postman collection](postman-tests/MALT-Api-Search-And-Add-Users.postman_collection.json), [postman environment](malt-local-env.postman_environment.json) and an external datafile with valid/invalid data to pass on to the backend requests.
```

File: maltd-frontend/package-lock.json (hunk 1 above)
Comment (karthikmsqa) [reply maltd-frontend/package-lock.json]: This seems like a reflection of .env file removal. I did not make any changes.

**Comment by karthikmsqa** [comment]
> .env and .env.example should not be deleted, please put them back

Added them back. Sorry about that.

**Comment by karthikmsqa** [comment]
> .env and .env.example should not be deleted, please put them back

Added the files. Thanks.

File: maltd-frontend/.env.example [hunk 3]
```diff
@@ -1,5 +1,6 @@
+
 REACT_APP_MALTD_API=<base_url_of_backend_api>
 REACT_APP_KEYCLOAK_REDIRECT_URI=<redirect_url>
 REACT_APP_KEYCLOAK_URL=<keycloak_url>
 REACT_APP_KEYCLOAK_REALM=<keycloak_realm>
-REACT_APP_KEYCLOAK_CLIENT_ID=<keycloak_client_id>
+REACT_APP_KEYCLOAK_CLIENT_ID=<keycloak_client_id>
```
Comment (sdevalapurkar) [inline maltd-frontend/.env.example]: remove newline on top of this file, and add newline at end of the file please

Comment (karthikmsqa) [reply maltd-frontend/.env.example]: Removed and added lines.

File: maltd-frontend/.env [hunk 4]
```diff
@@ -1 +1 @@
-SKIP_PREFLIGHT_CHECK=true
+SKIP_PREFLIGHT_CHECK=true
```
Comment (sdevalapurkar) [inline maltd-frontend/.env]: add newline

File: maltd-frontend/.env.example [hunk 5]
```diff
@@ -3,3 +3,4 @@ REACT_APP_KEYCLOAK_REDIRECT_URI=<redirect_url>
 REACT_APP_KEYCLOAK_URL=<keycloak_url>
 REACT_APP_KEYCLOAK_REALM=<keycloak_realm>
 REACT_APP_KEYCLOAK_CLIENT_ID=<keycloak_client_id>
+
```
Comment (sdevalapurkar) [inline maltd-frontend/.env.example]: extra newline here

File: maltd-frontend/.env (hunk 4 above)
Comment (karthikmsqa) [reply maltd-frontend/.env]: Added new line. Thanks.

File: maltd-frontend/.env.example (hunk 5 above)
Comment (karthikmsqa) [reply maltd-frontend/.env.example]: Removed extra new line.

**Pull request merged**